docs/content/guides/limiting-concurrency-in-data-pipelines.mdx

```diff
@@ -414,7 +414,13 @@ To specify limits on the number of ops/assets tagged with a particular concurren
 
 The concurrency key should match the name that the op/asset is tagged with. For example, if the op/asset is tagged with `dagster/concurrency_key: redshift`, then the concurrency key should be `redshift`.
 
```

Comment:
Maybe the new content should get its own heading? Right now it feels like it's in a weird state - is it part of the 'specifying concurrency' stuff right above it?

Maybe something like `#### Freeing up concurrency slots`?